site/en/docs/extensions/mv3/manifest/content_scripts/index.md

```diff
@@ -0,0 +1,262 @@
+      "include_globs": ["*nytimes.com/???s/*"],
+      "exclude_globs": ["*science*"],
+      "js": ["content-script.js"]
+    }
+  ],
+  ...
+}
+```
+
+<div class="switcher">
+{% Compare 'better', 'Matches' %}
+```text
+https://www.nytimes.com/arts/index.html
+https://.nytimes.com/jobs/index.html
+```
+{% endCompare %}
+
+{% Compare "worse", "Does not match" %}
+```text
+https://science.nytimes.com
+https://www.nytimes.com/jobs/business
+https://www.nytimes.com/science
+```
+{% endCompare %}
+</div>
+
+{% endDetails %}
+
+## Frames {: #frames }
+
+The `"all_frames"` field specifies if the content script should be injected into all frames matching the specified URL requirements. If set to `false` it will only inject into the topmost frame. It can be used along with `"match_about_blank"` to inject into an `about:blank` frame. 
+
+To inject into other frames like `data:`, `blob:`, and `filesystem:`, set the `"match_origin_as_fallback"` to `true`. For details, see [Injecting in related frames][cs-inject-frames]
+
+`"all_frames"` Boolean
+: _Optional_. Defaults to `false`, meaning that only the top frame is matched.If specified true, it will inject into all frames, even if the frame is not the topmost frame in the tab. Each frame is checked independently for URL requirements, it won't inject into child frames if the URL requirements are not met.
+
+`"match_about_blank"`- Boolean
+: _Optional_. Defaults to `false`. Whether the script should inject into an `about:blank` frame where the parent URL matches one of the patterns declared in `"matches"`.
+
+`"match_origin_as_fallback"` - Boolean
+: _Optional_. Defaults to `false`. Whether the script should inject in frames that were created by a matching origin, but whose URL or origin may not directly match the pattern. These include frames with different schemes, such as `about:`, `data:`, `blob:`, and `filesystem:`.
+
+## Run time and execution environment {: #world-timings }
+
+By default, content scripts are injected when the DOM is complete (`"document_idle"`), meaning all its resources have finished loading, and live in a private isolated execution environment that isn't accessible to the page or other extensions.
+
+[`"run_at"`][scripting-runat] - `document_start` | `document_end` | `document_idle`
+: _Optional_. Specifies when the script should be injected into the page. It corresponds with the loading stated of [Document.readyState][mdn-ready-state]. Either `"document_start"` (the DOM is still loading), `"document_end"` (other resources are still loading), or `"document_idle"`, which corresponds to complete (the DOM and page resources have finished loading).
+
+[`"world"`][scripting-world] - `ISOLATED` | `MAIN`
+: _Optional_. The JavaScript world for a script to execute within. Defaults to `"ISOLATED"`, which is the execution environment unique to the content script. Choosing `"MAIN"` world, means the script will share the execution environment with the host page's JavaScript.
```

Comment:
Can we link to https://developer.chrome.com/docs/extensions/mv3/content_scripts/#isolated_world?

Reply:
I already had it in the aside below. It looked a bit redundant, but I'll add it to see if it fits